go/inst/instance_dao.go

```diff
@@ -3184,6 +3186,8 @@ func PopulateGroupReplicationInformation(instance *Instance, db *sql.DB) error {
 		@@global.group_replication_single_primary_mode
 	FROM
 		performance_schema.replication_group_members
+	WHERE
+		MEMBER_STATE <> 'OFFLINE'
```

Comment:
just for consistency 
```suggestion
		MEMBER_STATE != 'OFFLINE'
```